src/solver/EulerSolver.cpp

```diff
@@ -391,3 +396,95 @@ void Solver::EulerSolver::updateAerodynamicCoefficients() {
 void Solver::EulerSolver::BroadCastCoeffs(std::vector<double> &vec) {
 	MPI_Bcast(&vec[0], 3, MPI_DOUBLE, 0, MPI_COMM_WORLD);
 }
+
+void Solver::EulerSolver::EulerExplicit() {
+	updateDeltaTime();
```

Comment:
Can residual smoothing be applied to Euler explicit?
```
		if (_config.getResidualSmoothing()) {
			smoothResiduals();
		}
```

Reply:
Yes